README.md

```diff
@@ -1,37 +1,38 @@
 ## Logz.io nodejs metrics sdk
+
 This topic includes instructions on how to send custom metrics to Logz.io from your Node.js application.
 
 The included example uses the [OpenTelemetry JS SDK](https://github.com/open-telemetry/opentelemetry-js) and its based on [OpenTelemetry exporter collector proto](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-exporter-collector-proto).
 
 **Before you begin, you'll need**:
 Node 8 or higher
 
-
 **Note** This project works best with logzio as metrics backend, but its compatible with all backends that support `prometheuesrmotewrite` format
 
 ## Quick start
 
 Install the package:
 
 ```
-npm install logzio-nodejs-metrics-sdk@0.2.1
+npm install logzio-nodejs-metrics-sdk@0.3.0
 ```
 
 Set the variables in the following code snippet:
 
-|Environment variable|Description|
-|---|---|
-|url|  The Logz.io Listener URL for for your region, configured to use port **8052** for http traffic, or port **8053** for https traffic. |
-|token| Your Logz.io Prometheus Metrics account token.  |
+| Environment variable | Description                                                                                                                         |
+| -------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| url                  | The Logz.io Listener URL for for your region, configured to use port **8052** for http traffic, or port **8053** for https traffic. |
```

Comment:
Please add an example
Something like
....or port **8053** for https traffic. For example - https://listener.logz.io:8053 ...